### DISCUSSION TITLE
changing models path to another folder outside of the frontend's main path?

### DISCUSSION THREAD

**Discussion by coldasicee** [vladmandic/sdnext#3137] [Q&A]
I have a bunch of models on another a1111 fork and I was trying to get the models straight from there, is it possible?
I tried changing the path on settings>system path>Base path where all models are stored
I changed it to 'X:\aistuff\otherfork\models' (without quotes) but the frontend didn't recognize them

**Comment by brknsoul** [comment]
Keep all your models in the one place and set the Environment Variable `SD_MODELSDIR` to `X:\location\of\models`

In the other fork (a1111, forge) you can use the command line args `--ckpt-dir`, `--lora-dir`, and `--embeddings-dir`

Alternative, you can simply create symlinks to each folder (but don't symlink the entire `models` folder.)

**Reply by vladmandic** [reply]
env var is fine, note there is also `--models-dir`
